test/manipulation.test.js

```diff
@@ -688,6 +688,313 @@ describe('manipulation', function () {
     });
   });
 
+  if (!getSchema().connector.replaceOrCreate) {
+    describe.skip('replaceOrCreate - not implemented', function(done) { done; });
+  } else {
+    describe('replaceOrCreate', function() {
+      var Post;
+      var ds = getSchema();
+      before(function() {
+        Post = ds.define('Post', {
+          title: { type: String, length: 255, index: true },
+          content: { type: String },
+          comments: [String]
+        });
+      });
+
+      it('without options on create (promise variant)', function(done) {
```

Comment:
the test name should read like a sentence, where "it" stands for the value from the describe block. At the moment, you have "it/replaceOrCreate with options on create (promise variant)".

The usual convention is to describe what happens and under what conditions. You have the conditions in the test name, but you are missing what is the expected outcome. 

Let's keep this simple and just prefix your test names with "works", i.e.

```js
it('works without options on create (promise variant)', ...);
```

Please apply this change to other test cases too.